src/features/exercises/components/MediaExercise.tsx

```diff
@@ -85,6 +85,7 @@ const MediaExercise = ({
         </span>
       )}
 
+      {/* eslint-disable sonarjs/todo-tag */}
       {/* TODO: allow multiple textfields if needed */}
       <label htmlFor={exercise.textFieldTitle} className="text-heading-3">
```

Comment:
Should we do something for this? I'm thinking that we could get rid of this as the tool should be already "complete".

Reply:
Removed the TODO.